### DISCUSSION TITLE
recipes/breads/baguettes/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1117] [Comments]
# recipes/breads/baguettes/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/baguettes/

**Comment by nicholaswilde** [comment]
- Be sure to score when dough is on the parchment 
- Be careful when transferring dough from couche.
- Careful when transferring parchment to baking stone.

**Reply by nicholaswilde** [reply]
- Be sure to score deep enough
